website/src/components/subscriptions/create/TriggerInput.tsx

```diff
@@ -7,6 +7,7 @@ import { BaselineInput } from './BaselineInput.tsx';
 import { VariantInput } from './VariantInput.tsx';
 import type { DateWindow } from '../../../types/DateWindow.ts';
 
+// TODO: Add latest version filter for WestNile #194
```

Comment:
Do we really need to do something here? I would have expected that we only need to change something in `FilterDisplay`.

Reply:
True. Initially, I thought to change the filters here to also store them in the backend. Now we don't store them. I moved this comment to `FilterDisplay`